tests/test_pv_creation.py

```diff
@@ -0,0 +1,77 @@
+"""
+A test for creating a PV
+"""
+import os
+import logging
+import ocs.defaults as defaults
+import yaml
+
+from ocs import exceptions
+from ocs import ocp
+from utility import utils
+from ocsci.enums import TestStatus
+
+
+log = logging.getLogger(__name__)
+
+PV_YAML = os.path.join("templates/ocs-deployment", "PersistentVolume.yaml")
+TEMP_YAML_FILE = 'test.yaml'
+
+RESOURCE = ocp.OCP(
+    kind='PersistentVolume', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+
+def create_pv(**kwargs):
```

Comment:
these basic functions (like create/delete pv) deserve to be moved to a common place

Reply:
of course but I think that for now, until we won't move to pytest, I prefer to keep things simpler for the test itself.
Could you please open a new issue for that and add me as an assignee?

Thanks

Reply:
@RazTamir https://github.com/red-hat-storage/ocs-ci/issues/50